docs/mpcnc/Pbase.md

```diff
@@ -0,0 +1,148 @@
+# Assembling The Base
+
+Click on the images to embiggen.
+
+## Legs
+![!MPCNC legs](https://www.v1engineering.com/wp-content/uploads/2020/06/Legs-scaled.jpg){: width="400"}
+
+* The leg lengths come from the [cut calculator](/calculator.md).
+* Do whatever it takes to get the legs as equal in length as possible. This will really help in leveling and keeping it level.
+* In this picture the one on the left is too short so it could either be shimmed during assembly or sand/grind the others.
+* A little extra time here pays off for the life of the machine.
+
+!!! Tip
+    Sand, grind, or file the inside and outside edges of all the cuts on the rails and legs. This prevents cut wires and eases assembly.
+
+___
+
+## Tools
+![!Tools Needed](https://www.v1engineering.com/wp-content/uploads/2020/06/foot-tools-scaled.jpg){: width="400"}
+
+* 8mm Socket or nut driver
+* Phillips #2 Screw driver
+___
+
+## Legs and Feet
+![!Parts and hardware](https://www.v1engineering.com/wp-content/uploads/2020/06/foot-ready-scaled.jpg){: width="400"}
+
+* You will need the legs, printed feet, M5x30 screws and locknuts.
+* It is best to pre-thread all the locknuts before assembly. This will loosen them and prevent mangaling of the printed parts.
+
+![!Assembling](https://www.v1engineering.com/wp-content/uploads/2020/06/foot-snug-scaled.jpg){: width="400"}
+
+* Insert the leg and make sure it is seated all the way to the table.
+* The feet have a small side for the head of the screw and a larger opening for the nut and socket.
+* Snug this screw. Snug means just a 1/4 turn past the nut making contact. This is does not need to be super tight. 
+* There is a 2+mm gap here, ***do not try to close the gap***.
+___
+
+## Base Layout
+![!Foot placement](https://www.v1engineering.com/wp-content/uploads/2020/06/base-layout-scaled.jpg){: width="400"}
+
+* Use whatever screws are appropreate for your table, these are not included in the V1 Kits.
+* Layout the foot assemblies on your table acording to the dimensions from the [cut calculator](/calculator.md).
+* Notice all the clamping screws are oriented towards the center of the build area.
+* Screw down the two assemblies along the front edge of the table. These will be your fixed references.
+
+![!Square check](https://www.v1engineering.com/wp-content/uploads/2020/06/base-check1-scaled.jpg){: width="400"}
+ 
+* Measure as carefully and accurately as you can across the leg tops. Your machine can only be as accurate as you build it.
+* Compare this measurement to the next one.
+
+![!Compare measurements](https://www.v1engineering.com/wp-content/uploads/2020/06/base-check-2-scaled.jpg){: width="400"}
+
+* This next diagonal should match the previous one. Squaring this way can be extremely accurate as the deviation is multiplied.
```

Comment:
```suggestion
* This next diagonal should match the previous one. Squaring this way can be extremely accurate because any error in the leg locations is multiplied in this measurement.
```